.circleci/config.yml

```diff
@@ -14,7 +53,184 @@ jobs:
             flake8 .
             mypy .
 
+  slack_notify:
+    parameters:
+      message:
+        type: string
+        default: ":tada:"
+      execute:
+        type: boolean
+        default: true
+    docker:
+      - image: 'cimg/base:stable'
+    steps:
+      - when:
+          condition: << parameters.execute >>
+          steps:
+            - slack/notify:
+                custom: |
+                  {
+                    "blocks": [
+                      {
+                        "type": "section",
+                        "fields": [
+                          {
+                            "type": "plain_text",
+                            "text": "<< parameters.message >>",
+                            "emoji": true
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                event: always
+                channel: $SLACK_SDK_NIGHTLY_CI_CHANNEL
+      # this is to make sure `steps` is not empty
+      - run:
+          name: Print message to stdout
+          command: echo << parameters.message >>
+
+  create_vm_gcp:
+    parameters:
+      framework:
+        type: string
+      execute:
+        type: boolean
+        default: true
+    docker:
+      - image: "python:3.8"
+    steps:
+      - checkout
+      - when:
+          condition: << parameters.execute >>
+          steps:
+            - gke/install
+#            - run:
```

Comment:
Do you want to keep commented out code?

Reply:
yea..it might eventually stop working if the shipped gcloud version gets too outdated, and then i'll just uncomment this block :-)